.github/workflows/ci.yml

```diff
@@ -21,6 +21,9 @@ jobs:
           - NodeVersion: 22.19.x
             NodeVersionDisplayName: 22
             OS: ubuntu-latest
+          - NodeVersion: 24.9.x
+            NodeVersionDisplayName: 24
+            OS: ubuntu-latest
           - NodeVersion: 22.19.x
```

Comment:
The same warning is also printed for this code in `_executeLifecycleCommandInternal()`:

https://github.com/microsoft/rushstack/blob/fb6faf2f9a6cefe72802c7ca8edb13054cdec783/libraries/rush-lib/src/utilities/Utilities.ts#L682

However in this case the API is not being used correctly:
```
let commandFlags: string = '/d /s /c';   🤦‍♂️
```